lib/ci/reporter/test_unit/version.rb

```diff
@@ -1,7 +1,7 @@
 module CI
   module Reporter
     module TestUnitVersion
-      VERSION = "1.0.0"
```

Comment:
Please don't change this in this PR. We will need to update it, certainly, but that's better done as part of the release process.